sdks/go/pkg/beam/io/healthcare/gcpcommon.go

```diff
@@ -0,0 +1,34 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package healthcare
+
+import (
+	"context"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/core"
+	healthcarex "google.golang.org/api/healthcare/v1"
+	"google.golang.org/api/option"
+)
+
+const userAgent = "apache-beam-io-google-cloud-platform-healthcare/" + core.SdkVersion
+
+func NewGcpHealthcareService() *healthcarex.Service {
```

Comment:
Question: Is this function intended to be user-facing? Or is it only exported for developer convenience? If it's the latter, we should hide it from users by putting it in an `internal/`subdirectory. [More info](https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface)

Reply:
Yes, it was only being exported for developer convenience. I wasn't familiar with the special `internal` directory, thanks for sharing.
I ended up getting rid of this file completely and simply moving its content to `common.go` inside `fhirio` package. I had created this file because I was anticipating the creation of the other healthcare IO connectors (i.e. [DicomIO](https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIO.java), [HL7v2IO](https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IO.java)), but I figured it's better to worry about them once we actually decide to implement them in Go.